rosdep/base.yaml

```diff
@@ -1496,13 +1496,27 @@ libbluetooth-dev:
   fedora: [bluez-libs-devel]
   gentoo: [net-wireless/bluez-libs]
   ubuntu: [libbluetooth-dev]
+libboost-program-options:
+  debian:
+    jessie: [libboost-program-options1.55.0]
+    stretch: [libboost-program-options1.62.0]
+  fedora: [boost-program-options]
+  ubuntu:
+    bionic: [libboost-program-options1.65.1]
+    trusty: [libboost-program-options1.55.0]
+    xenial: [libboost-program-options1.58.0]
+libboost-program-options-dev:
+  debian: [libboost-program-options-dev]
+  fedora: [boost-program-options]
```

Comment:
This should be [boost-devel](https://apps.fedoraproject.org/packages/boost-devel) for Fedora.

Reply:
> This should be boost-devel for Fedora.

Is this because boost development headers aren't split into individual packages, so any -dev boost package must depend on boost-devel?